arcgis-ios-sdk-samples/Analysis/Distance measurement analysis/README.md

```diff
@@ -0,0 +1,25 @@
+# Distance Measurement Analysis
+This sample demonstrates measuring 3D distances between two points in a scene.
+
+The distance measurement analysis allows you to add the same measuring experience found in ArcGIS Pro, City Engine, and the ArcGIS API for JavaScript to your app. You can set the unit system of measurement (metric or imperial) and have the units automatically switch to one appropriate for the current scale. The rendering is handled internally so they do not interfere with other analyses like viewsheds.
+
+![Image](image1.png)
+
+## How to use the sample
+Choose a unit system for the measurement in the segmented control. Click any location in the scene to start measuring. Move the mouse to an end location, and click to complete the measure. Click a new location to clear and start a new measurement.
```

Comment:
This references how the macOS version works. I think this is more of a tap-drag workflow, and we should not mention 🐭.

Reply:
Thank you. I changed the first sentence, but completely ignored the rest of the paragraph. I'll update it.